examples/simple-sam/src/main/java/ca/ryangreen/examples/Main.java

```diff
@@ -36,10 +42,24 @@ public void hello(InputStream inputStream, OutputStream outputStream, Context co
     // this function is exposed by /sdk-example and is unauthenticated
     public void handleRequest(InputStream inputStream, OutputStream outputStream, Context context) throws IOException {
 
+        String apiIntputStream = new Scanner(inputStream).useDelimiter("\\A").next();
+        JsonFactory factory = new JsonFactory();
+        ObjectMapper mapper = new ObjectMapper(factory);
+        JsonNode rootNode = mapper.readTree(apiIntputStream);
+        String myApiId = rootNode.path("requestContext").path("apiId").asText();
```

Comment:
Can we give this a default value like "TODO" in case this is not called by API Gateway